### ISSUE TITLE
;handle pretty <string> is showing following error on terminal

### ISSUE THREAD

**Issue by svaderia** [cheran-senthil/TLE#320]
This is a wrong use of the command, but I didn't know only integers are allowed (help is a little vague). So should it throw the exception on the screen? 
```
Traceback (most recent call last):
  File "/Users/svaderia/Library/Caches/pypoetry/virtualenvs/tle-d5qC4QT3-py3.7/lib/python3.7/site-packages/discord/ext/commands/core.py", line 370, in _actual_conversion
    return converter(argument)
ValueError: invalid literal for int() with base 10: 'svaderia'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/svaderia/Library/Caches/pypoetry/virtualenvs/tle-d5qC4QT3-py3.7/lib/python3.7/site-packages/discord/ext/commands/bot.py", line 863, in invoke
    await ctx.command.invoke(ctx)
  File "/Users/svaderia/Library/Caches/pypoetry/virtualenvs/tle-d5qC4QT3-py3.7/lib/python3.7/site-packages/discord/ext/commands/core.py", line 1158, in invoke
    await ctx.invoked_subcommand.invoke(ctx)
  File "/Users/svaderia/Library/Caches/pypoetry/virtualenvs/tle-d5qC4QT3-py3.7/lib/python3.7/site-packages/discord/ext/commands/core.py", line 721, in invoke
    await self.prepare(ctx)
  File "/Users/svaderia/Library/Caches/pypoetry/virtualenvs/tle-d5qC4QT3-py3.7/lib/python3.7/site-packages/discord/ext/commands/core.py", line 685, in prepare
    await self._parse_arguments(ctx)
  File "/Users/svaderia/Library/Caches/pypoetry/virtualenvs/tle-d5qC4QT3-py3.7/lib/python3.7/site-packages/discord/ext/commands/core.py", line 599, in _parse_arguments
    transformed = await self.transform(ctx, param)
  File "/Users/svaderia/Library/Caches/pypoetry/virtualenvs/tle-d5qC4QT3-py3.7/lib/python3.7/site-packages/discord/ext/commands/core.py", line 455, in transform
    return await self.do_conversion(ctx, converter, argument, param)
  File "/Users/svaderia/Library/Caches/pypoetry/virtualenvs/tle-d5qC4QT3-py3.7/lib/python3.7/site-packages/discord/ext/commands/core.py", line 408, in do_conversion
    return await self._actual_conversion(ctx, converter, argument, param)
  File "/Users/svaderia/Library/Caches/pypoetry/virtualenvs/tle-d5qC4QT3-py3.7/lib/python3.7/site-packages/discord/ext/commands/core.py", line 379, in _actual_conversion
    raise BadArgument('Converting to "{}" failed for parameter "{}".'.format(name, param.name)) from exc
```

**Comment by goswami-rahul** [comment]
Right, it show usage probably. I think a better way would be make it intended bahaviour when searching with handles, so it returns the page with the handle on it.

**Comment by krofna** [comment]
fixed in de3e4d516d6be529a5ad899f546ea95092538fde
